test.py

```diff
@@ -127,6 +127,7 @@
     'small_vector_test',
     'data_listeners_test',
     'truncation_migration_test',
+    'like_matcher_test'
 ]
```

Comment:
End with a trailing comma to make mix-and-match easier.

Reply:
Done.